services/postgresql-docker.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "postgresql-docker",
+  "description": "PostgreSQL running in a Docker container (postgresql)",
+  "dependencies": {
+    "provides": [ "postgresql", "database" ],
```

Comment:
Should this `postgresql` match the image_name `postgres`? Just wondering, as I am not sure which of these are descriptions/display names or actual values that *need* to be something.